src/test/java/seedu/address/logic/commands/AddCommandTest.java

```diff
@@ -155,7 +155,8 @@ public ObservableList<Person> getFilteredPersonList() {
 
         @Override
         public void updateFilteredPersonList(Predicate<Person> predicate) {
-            throw new AssertionError("This method should not be called.");
```

Comment:
Quoting the header comment, this is "a default model stub that has all of the methods failing". I'm not sure if this should be done. 
If I'm not wrong, subclasses of this class need to override a method if it wants to allow that method to be called. In this case, I believe the `updateFilteredPersonList` method can be overridden in the `ModelStubAcceptingPersonAdded` class to allow the tests to work as intended (not 100% sure about this).

Reply:
I think what you are suggesting is more correct regardless, will make the changes 👍